### PR TITLE
mentioned QuDiffEq.jl

### PR DESCRIPTION
**Pull request by dgan181**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
It should go in the ODE solver page instead.

**Comment by dgan181** [comment]
Yeah. Sorry. I didn't look at the message carefully. So is this fine or should I add an example?

**Comment by ChrisRackauckas** [comment]
Take a look at this page. Make a section after this one http://docs.juliadiffeq.org/latest/solvers/ode_solve.html#TaylorIntegration.jl-1 that introduces the new algorithm choices and its options.

**Comment by dgan181** [comment]
Ah. So I just include the last method (the one that uses `::ODEProblem`) then? 
The linearised form for HHL can be developed just as simply as for the other one. I was planning to add that right after fixing some implementation issues in HHL. I was having trouble with the getting the correct order of precision in higher order methods. I still need to figure that out.
FYI,  HHL isn't a great way to go about solving differential equations because circuit becomes increasingly bulky with more dimensions (variables and time steps).

**Comment by ChrisRackauckas** [comment]
>Ah. So I just include the last method (the one that uses ::ODEProblem) then?

Yup, or add the HHL ones if they are going to be done soon.

File: docs/src/solvers/ode_solve.md [hunk 1]
```diff
@@ -1047,6 +1047,22 @@ using TaylorIntegration
 
 Note: this method is much faster if you put `@taylorize` on your derivative function!
 
+  ### QuDiffEq.jl
```
Comment (ChrisRackauckas) [inline docs/src/solvers/ode_solve.md]:
```suggestion
### QuDiffEq.jl
```

**Pull request merged**